src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -111,9 +111,14 @@ function getAjaxStudentList(ajaxPreloadImage) {
 /**
  * Handles how the panels are displayed, including rendering the spreadsheet interface.
  */
-function adjustStudentsPanelView(panelHeading, panelCollapse) {
+function adjustStudentsPanelView(panelHeading, panelCollapse, isDataHandsontable) {
     toggleStudentsPanel(panelHeading, panelCollapse);
-    dataHandsontable.render(); // needed as the view is buggy after collapsing the panel
+    // needed as the view is buggy after collapsing the panel
+    if (isDataHandsontable) {
+        dataHandsontable.render();
+    } else {
+        enrollHandsontable.render();
```

Comment:
better to just pass the `Handsontable` instead of using a boolean argument like this.

Reply:
Also, the name is not very nice.

Reply:
Turns out I cannot pass the `Handsontable` as a variable and then call `.render()` on it. It throws something like `Uncaught TypeError: handsontableInstance.render is not a function` in the developers tools. Would keeping the boolean argument but renaming be better?